### ISSUE TITLE
Create a plan for scanning character from _some_ CMIP5 data

### ISSUE THREAD

**Issue by agstephens** [roocs/34e-mngmt#8]
Draw a diagram of the software architecture needed to scan lots of CMIP5 ESGF data sets (in parallel on LOTUS), and capture the character in an appropriate format/database.

Capture and log any exceptions when we cannot read a dataset and/or cannot capture a specific characteristic.

**Comment by ellesmith88** [comment]
https://docs.google.com/document/d/16AuSC8dKgIc_rmrwFsyRbHY6NKhfiTh7c_EaNFzZ8sQ/edit?usp=sharing

**Comment by agstephens** [comment]
Hi @ellesmith88, the document is a good start. As a next step, please can you:
 1. List the _characteristics_ that you think we will be capturing
 2. Add a mention of how we will record the character information.

For (2), I think it makes sense to start with a JSON file per Dataset. In a directory structure such as:
`${OUTPUT_DIR}/<drs_level_1>/<drs_level_2>/<drs_level_3>/<drs_level_4>/<drs_level_5>/<full_drs>.json`

E.g. 
`outputs/cmip5/output1/MOHC/HadGEM2-ES/rcp45/cmip5.output1.MOHC.HadGEM2-ES.rcp45.mon.atmos.Amon.r1i1ip1.tas.v20130122`
(NOTE: I haven't checked this is a completely valid DRS entry - it's just an example).

**Comment by agstephens** [comment]
Superseded by other issues.
